cmake/Modules/Findaom.cmake

```diff
@@ -42,9 +42,21 @@ mark_as_advanced(AOM_INCLUDE_DIR AOM_LIBRARY AOM_LIBRARIES)
 if(AOM_LIBRARY)
     if("${AOM_LIBRARY}" MATCHES "\\${CMAKE_STATIC_LIBRARY_SUFFIX}$")
         add_library(aom STATIC IMPORTED GLOBAL)
+        set(_AOM_PC_PREFIX "_AOM_STATIC")
     else()
         add_library(aom SHARED IMPORTED GLOBAL)
+        set(_AOM_PC_PREFIX "_AOM")
```

Comment:
We should only need the new code when the aom library is a static library. Can we move the new code into the if block (for the static library case) above, at line 45?